### ISSUE TITLE
Typo in WCAG Bronze section of Jan 6 editor's draft

### ISSUE THREAD

**Issue by mraccess77** [w3c/silver#228]
"Bronze level means that the content in scope does have any critical errors and meets the minimum percentage"

Should be

"Bronze level means that the content in scope does not have any critical errors and meets the minimum percentage"

**Comment by jspellman** [comment]
Thank you for your response, we have implemented your suggestions for the next draft. We appreciate the pull request, but because of W3C rules on intellectual property, we can't accept a PR from people who haven't joined the working group or community group.

**Comment by rachaelbradley** [comment]
This has been resolved in the newest draft. The exact sentence is no longer included.

**Comment by rachaelbradley** [comment]
This has been captured in our core quesitons list.
